ingester/src/init/wal_replay.rs

```diff
@@ -314,6 +315,10 @@ mod tests {
         }
     }
 
+    lazy_static! {
+        static ref ALTERNATIVE_TABLE_NAME: &'static str = "arán";
+    }
+
```

Comment:
I believe this can just be:

```suggestion
    const ALTERNATIVE_TABLE_NAME: &str = "arán";
```

`lazy_static` is only needed if you have to do some non-const operation to construct the value.